playbooks/roles/deploy-osh/templates/neutron.yaml.j2

```diff
@@ -1 +1,23 @@
-{{ lookup('template','files/common-images-overrides.yml') | from_yaml | to_nice_yaml(indent=2) }}
\ No newline at end of file
+{% if developer_mode %}
```

Comment:
We should list here all the upstream patches necessary, for helping the other squad, and tracking.

Reply:
There's no current upstream patch for neutron.

Besides, I do not think we should communicate via comments in the code. Also we should not use code for tracking bugs and TODOs